docs/RBC_EN.md

```diff
@@ -0,0 +1,52 @@
+# RBC : Reliable Broadcast
+
+
+
+### overview
+
+The typical Broadcast, which depends on implementation, does not guarantee message delivery and does not consider situations in which the sender is crash (the situation in which the message cannot be sent to all). However, the RBC guarantees message delivery and partially solves the situation where the sender crashes. And the Atomic Broadcast introduced in this [paper](<https://eprint.iacr.org/2016/199.pdf>) is a Broadcast that guarantees the situation of the sender crash and guarantees the order of messages, and the RBC to be introduced is Atomic Broadcast.
+
+
+
+[The paper](<https://eprint.iacr.org/2016/199.pdf>) shows ABC (Atomic Broadcast) as ACS (Asynchronous Common Subset) using `threshold encryption`and ACS consists of RBC and BBA (Binary Byzantine Agreement) components. Among them, RBC performs reliable broadcasting and BBA is responsible for ordering of consensus and atomic messages.
+
+
+
+HBBFT does not send the entire data but sends the pieces by dividing the data by the number of nodes in the network. For this purpose, RBC uses a technique called `erasure coding`.
+
+### Erasure coding   
+
+Erasure coding is one of FEC (Forward Error Correction) methods and it can perform error detection and error correction. In RBC, Reed-solomon technique is used among erasure coding techniques. It splits the original data into n pieces and adds k parity. Then, even if a maximum of k data is lost, original data can be recovered if only n data are survived. Good Examples and detailed explanations can be found [here](<https://www.backblaze.com/blog/reed-solomon/>).
```

Comment:
In `Then, even if a maximum of k data is lost, original data can be recovered if only n data are survived.`, is it right `n pieces`? I think that it should be `(n-k) pieces`.

Reply:
@AgwaB 
Sorry, I was confused. You're right :)